connectivity/check/features.go

```diff
@@ -473,6 +485,34 @@ func (ct *ConnectivityTest) ForceDisableFeature(feature Feature) {
 	ct.features[feature] = FeatureStatus{Enabled: false}
 }
 
+// isFeatureKNPEnabled checks if the Kubernetes Network Policy feature is enabled from the configuration.
+// Note that the flag appears in Cilium version 1.14, before that it was unable even thought KNPs were present.
```

Comment:
Is there a PR or issue somewhere tracking this? I don't see the flag in the main branch of cilium/cilium. (I'd be completely fine with just replacing the internals of this function with `return true` then if we deem it useful to add a flag to cilium/cilium, follow up on that)

Reply:
I am referring to the following flag in Cilium:
https://github.com/cilium/cilium/blob/87c8746e254d2b3ccdf8a42bb48367b6ecf2890f/pkg/defaults/defaults.go#L521
Maybe my comment is not clear... 🤨